src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -474,42 +371,56 @@ void spacetime_deriv_damped_harmonic_h(
   }
 }
 
-// Explicit Instantiations
-/// \cond
+template <size_t SpatialDim, typename Frame>
+void DampedHarmonicCompute<SpatialDim, Frame>::function(
+    const gsl::not_null<return_type*> h_and_d4_h,
+    const db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>& gauge_h_init,
+    const db::item_type<Tags::SpacetimeDerivInitialGaugeH<SpatialDim, Frame>>&
+        dgauge_h_init,
+    const Scalar<DataVector>& lapse,
+    const tnsr::I<DataVector, SpatialDim, Frame>& shift,
+    const tnsr::a<DataVector, SpatialDim, Frame>&
+        spacetime_unit_normal_one_form,
+    const Scalar<DataVector>& sqrt_det_spatial_metric,
+    const tnsr::II<DataVector, SpatialDim, Frame>& inverse_spatial_metric,
+    const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
+    const tnsr::aa<DataVector, SpatialDim, Frame>& pi,
+    const tnsr::iaa<DataVector, SpatialDim, Frame>& phi, const double time,
+    const double t_start, const double sigma_t,
+    const tnsr::I<DataVector, SpatialDim, Frame>& coords,
+    const double sigma_r) noexcept {
+  if (UNLIKELY(h_and_d4_h->number_of_grid_points() != get(lapse).size())) {
+    h_and_d4_h->initialize(get(lapse).size());
+  }
+  damped_harmonic(
+      make_not_null(&get<Tags::GaugeH<SpatialDim, Frame>>(*h_and_d4_h)),
+      make_not_null(
+          &get<Tags::SpacetimeDerivGaugeH<SpatialDim, Frame>>(*h_and_d4_h)),
+      gauge_h_init, dgauge_h_init, lapse, shift, spacetime_unit_normal_one_form,
+      sqrt_det_spatial_metric, inverse_spatial_metric, spacetime_metric, pi,
+      phi, time, coords, 1., 1.,
+      1.,                // amp_coef_{L1, L2, S}
+      4, 4, 4,           // exp_{L1, L2, S}
```

Comment:
Maybe find a way to not hard-code the same magic numbers multiple times? Or maybe that's too pedantic, and it's fine...just a suggestion

Reply:
Since this is such a small change I squashed it in. I made them `constexpr`. I agree we should make it an input file option, but I'd like to get all the DH changes I currently have on a branch in before doing that :)